v2/cmd/asoctl/pkg/importing/progress.go

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package importing
```

Comment:
I had the same conversation with myself.

Reply:
I'll give this some thought and try to find a better name for the package - at the moment its internal and doesn't really matter, but after this PR it will be part of the API so we should really try and get it right.

Reply:
I renamed the package to `resourceimporter`.